run_tests.sh

```diff
@@ -22,6 +22,25 @@ set -ex
 # Default GOVERSION
 [[ ! "$GOVERSION" ]] && GOVERSION=1.11
 REPO=dcrdata
+ROOTMODULE="github.com/decred/dcrdata"
+ALLMODULES="$ROOTMODULE/api/types
+  $ROOTMODULE/blockdata
+  $ROOTMODULE/db/cache
+  $ROOTMODULE/db/dbtypes
+  $ROOTMODULE/db/dcrpg
+  $ROOTMODULE/db/dcrsqlite
+  $ROOTMODULE/exchanges
+  $ROOTMODULE/explorer/types
+  $ROOTMODULE/gov/agendas
+  $ROOTMODULE/gov/politeia
+  $ROOTMODULE/mempool
+  $ROOTMODULE/middleware
+  $ROOTMODULE/pubsub
+  $ROOTMODULE/pubsub/types
+  $ROOTMODULE/rpcutils
+  $ROOTMODULE/semver
+  $ROOTMODULE/stakedb
+  $ROOTMODULE/txhelpers"
```

Comment:
seems like there should be a way to get this list with `go list -m`, I can't a syntax that gets it I suppose this list is ok.  @jrick Know a way to handle this testing other than manually listing the modules?

Reply:
since the concept of multiple modules in a single project is relatively new, I suspect that such a solution doesn't exist yet. Anyway, I am doing some wide research on the same to see what progress in being made by the go dev team.

Reply:
This was already merged, and I'm just seeing it now, but you could do like [dcrd](https://github.com/decred/dcrd/blob/6dbd893/run_tests.sh#L43-L52) or [dcrwallet](https://github.com/decred/dcrwallet/blob/master/run_tests.sh) does.

Reply:
Sure. Noted, I will add the change.

Reply:
I had tried `'{{.Path}}'` instead of `'{{.Dir}}'` and ended up with the old `v3` module getting listed too.  `'{{.Dir}}'` solves it.  Thanks!